src/components/PerfherderGraph/index.jsx

```diff
@@ -91,6 +97,10 @@ class PerferhderGraph extends React.Component {
         if (!chartJsOptions) {
           chartJsOptions = generateChartJsOptions(meta, yLabel);
         }
+        chartJsOptions.scales.xAxes[0].ticks = {
```

Comment:
I think `minDate`, `maxDate` should be passed to `generateChartJsOptions` and this `chartJsOptions.scales.xAxes[0].ticks` change should be done inside `generateChartJsOptions`

This tests will need some changes after this